pkg/plugins/util/cleanup.go

```diff
@@ -76,12 +76,12 @@ func UpdateKustomizationsInit() error {
 		`
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
-#- manager_webhook_patch.yaml
+#- path: manager_webhook_patch.yaml
```

Comment:
patchesStrategicMerge was changed to patches upstream, causing these changes: https://github.com/kubernetes-sigs/kubebuilder/pull/3661